### PR TITLE
A start at a tutorial notebook for SPD 2019

### PR DESCRIPTION
**Pull request by ehsteve**
The plan is also to use part of this or a shorter version for the SunPy talk at the 2019 PyHC meeting.

### CODE REVIEW

**Comment by ehsteve** [comment]
ping @hayesla

File: 2019_spd/sunpy-1p0.ipynb [hunk 1]
```diff
@@ -0,0 +1,441 @@
...
...5xrQGWFJYZoNNWAgGhEYQS1SEZznD37D9XdfMckxXn5Ph4PcFs2/6eHxkKuLaaVfYB/F35i8/ySXX9nmFIajI+PtnlP0Jfnqq58gfWL1mGpHRN/FMl+P+g5UUg+a9K+g9ardbfarVaX/sMzvWlKT+vpEsZtC2T1hoYPKerCzkQnQAMteXkVAgqUCwYAIbC/v4b1lxtIXk8ooOQW4ydTqcxELpF6WErHrIBKDt4uHDAFb2Aea3Evc7X6P8x7tVCGYoDp5SS/EvlOnyo65h97yksAILCa0EV1ueBAlCHak5EoJtoXygiQoBmyrAf9/86S8BANVbOqGm7jpqhKOzHZ0CNQW2sHOABStlvHh7DgLUePDudRjvWYTdSTjZctU38W0/SW4fSnXcKzTl+CW3fxlC6/SmiHi+jB9J3f1dSribCosvtdliC68w+ZZt5SfS+mAe1ev44fJHf/0783h9KJw8izIMJxck4/WDct4fKgf5EGSdKOVb2Max7wAVxCZMKZ0J6yrR23H/cCZ7eDlD0VHFYdjAL3NelEmz4T2q6J7iuHUk3Op01QzBTM7wKwMeS6yljkH1yiO/xVEF58sw9sXrQXp4gAMBEpEObMCnwtHY+NpJfgkkD/RmwPKy2/0HKJ4LkarX67mq1+mVJvyHp/261Wv9Hq9X6G7w+g3O/1uVMeQNcuYll5f4tRDxsQ+C9lDcS8ER1OlU6z5lx1opSrERWDWHgZiD0m0vdOMYzNZN1A7b7+/uNsJLrQh54IP2aPU6QbXmIpMxt6v/dsH1ox+tmqq4ylV07s+jaB0PSASkfUB7KOqYL0cGt6hx+rYgTeNWFtujZd3/3CQWWI20IUFKvp8pQCsQMvuByrTT1gfBSEa7QbsfrsozKy0W8+q3cr6bHNvb3GwPqjpJOBbC3lAAJgLt1dPedksgcfuzTLMk/SmGFkKMDiRVBTqUnvytNzsMveWMYCzRLksZqJFUYn4cVOS9ASHaihgp4HC+SPUC1uiU4ikOv/V5ThRgMCw6wcHHLsOx/oAwPq8Ve3idZKcYVsrQ9fQSrS0qLklU/0Eq4hefhHLgQAMaN/f2173JfCYao6qUEKtirHTUTE1yn0QL0AT1nJkiawDV52ApUMOdwrsH7KGJJZ4SInaQNLyT9in6w8sIQkFar9WOS/qKkvyvp1/XZULyvfQEgXZzDslXue6SxfM
```
Comment (hayesla) [inline 2019_spd/sunpy-1p0.ipynb]: This plot is off at the moment - you need to add np.deg2dat(this_coord.lon)

**Comment by hayesla** [comment]
Looks good Steven - I think we should add in 

* using unified downloader Fido
* different data types in SunPy - map and timeseries
* examples of working with these data types - manipulation, management of transforms

File: 2019_spd/sunpy-1p0.ipynb (hunk 1 above)
Comment (ehsteve) [reply 2019_spd/sunpy-1p0.ipynb]: Thanks!

**Comment by ehsteve** [comment]
@hayesla those sound like good ideas though let's make sure to focus on the new stuff.

**Comment by Cadair** [comment]
Do we really want to "focus on the new stuff"? Is this a tutorial on what's new in SunPy or "why you should convert from IDL"?

**Comment by ehsteve** [comment]
I was thinking it would be a SunPy 1.0 talk. The end result should be "I should convert to Python"!

**Comment by Cadair** [comment]
I feel like we might need to do a bit of background entry-level sunpy as well.

**Comment by ehsteve** [comment]
Yeah, I think that's what @hayesla is saying.

**Comment by ehsteve** [comment]
What's with this scary delete fork button that appears now? Ugh.

**Comment by Cadair** [comment]
Yeah it's terrible :woman_facepalming:

**Pull request merged**